newrelic/hooks/mlmodel_openai.py

```diff
@@ -559,13 +582,15 @@ def _record_completion_error(transaction, linking_metadata, completion_id, kwarg
     output_message_list = []
     if "content" in kwargs:
         output_message_list = [{"content": kwargs.get("content"), "role": kwargs.get("role")}]
+
```

Comment:
```suggestion
```

Reply:
I had intentionally added that newline in for readability to separate out the call to `create_chat_completion_message_event` a bit more 😄